test/controllers/SingleRecordControllerSpec.scala

```diff
@@ -239,5 +248,298 @@ class SingleRecordControllerSpec extends SpecBase with MockitoSugar {
       }
     }
 
+    "TraderReferenceSummary.row" - {
```

Comment:
These tests are in the wrong place?

Reply:
No, they are in right places. as we display these rows on singlerecord page